documentation/docs/80-adapter-api.md

```diff
@@ -26,6 +26,7 @@ The types for `Adapter` and its parameters are available in [types/config.d.ts](
 Within the `adapt` method, there are a number of things that an adapter should do:
 
 - Clear out the build directory
+- Call `utils.update_ignores` to ignore build output in existing `.gitignore`, `.eslintignore` and `.prettierignore` files at the root of the project
```

Comment:
My nit on this is that "root of the project" and the location of `svelte.config.js` can be different things, hence why I asked the clarifying question in the first place.

Reply:
:pray: